packages/components/RadioGroup/src/RadioButton.settings.ts

```diff
@@ -60,7 +60,7 @@ export const settings: IComposeSettings<IRadioButtonType> = [
     content: {
       variant: 'subheaderStandard',
       style: {
-        marginTop: 3,
+        marginTop: 2,
```

Comment:
why isn't this just 0? If we're centering it, we wouldn't need an offset?

Reply:
I think when I played with this in NetUI spy, the "O" of "Option" looked slightly north of center when the top margin is 0, so it looked off-center the other way. But Sammy can post a picture here and we can take a look.

Reply:
I've tested it with all values/DPIs, and this seemed to work best. I believe '0' doesn't work bet because the variant of the font + the size of it's border. Although the border is invisible, it still exists and the size is taken into account when centering